specs/clickhouse.spec

```diff
@@ -73,13 +75,13 @@ Source100:         checksum.sha512
 
 BuildRoot:         %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 
-BuildRequires:     devtoolset-9-gcc-c++ devtoolset-9-binutils
+BuildRequires:     devtoolset-10-gcc-c++ devtoolset-10-binutils
 BuildRequires:     cmake3 openssl-devel libicu-devel libtool-ltdl-devel
-BuildRequires:     unixODBC-devel readline-devel
+BuildRequires:     unixODBC-devel readline-devel librdkafka-devel lz4-devel
```

Comment:
@gongled Can you please explain why did you return `librdkafka` and `lz4` to the required dependencies? Later in `%build` section, internal versions of these libraries are used for the build.

Reply:
I had experimented with dynamic linkage to `librdkafka` and `lz4`, and I just forgot to prune these dependencies from `BuildRequires`. ClickHouse uses their own "vendored" and patched dependencies, so I do not see any reasons not to link them statically as is.